kernel/src/actions/mod.rs

```diff
@@ -16,23 +12,30 @@ use crate::actions::schemas::GetStructField;
 use crate::features::{ReaderFeatures, WriterFeatures};
 use crate::{schema::StructType, DeltaResult, EngineData};
 
+pub mod deletion_vector;
+pub mod set_transaction;
+
+pub(crate) mod schemas;
+pub(crate) mod visitors;
+
 pub(crate) const ADD_NAME: &str = "add";
 pub(crate) const REMOVE_NAME: &str = "remove";
 pub(crate) const METADATA_NAME: &str = "metaData";
 pub(crate) const PROTOCOL_NAME: &str = "protocol";
 pub(crate) const TRANSACTION_NAME: &str = "txn";
```

Comment:
```suggestion
pub(crate) const SET_TRANSACTION_NAME: &str = "txn";
```

Will need to fix below too

Reply:
great catch thanks